### PR TITLE
Cran update

### PR DESCRIPTION
**Pull request by SvenVw**
The PR contains no changes to the calculations and only contains small fixes for submission

### CODE REVIEW

**Comment by BrentWHR** [comment]
If, according to CRAN, the function examples take too long to run, I think we can add dontrun tags as:
````
#' @examples
#' \dontrun{
#' 
#' examples go here
#' 
#' }
````

**Comment by SvenVw** [comment]
Feedback from CRAN gives still 3 NOTES:

```
Flavor: r-devel-linux-x86_64-debian-gcc, r-devel-windows-x86_64
Check: CRAN incoming feasibility, Result: Note_to_CRAN_maintainers
  Maintainer: 'Sven Verweij <sven.verweij@nmi-agro.nl>'

Flavor: r-devel-linux-x86_64-debian-gcc
Check: examples, Result: NOTE
  Examples with CPU time > 2.5 times elapsed time
                         user system elapsed ratio
  calc_sbal_arable      1.640  0.036   0.190 8.821
  calc_waterstressindex 1.079  0.155   0.219 5.635

Flavor: r-devel-linux-x86_64-debian-gcc
Check: tests, Result: NOTE
    Running 'testthat.R' [62s/8s]
  Running R code in 'testthat.R' had CPU time 7.8 times elapsed time

Flavor: r-devel-linux-x86_64-debian-gcc
Check: re-building of vignette outputs, Result: NOTE
  Re-building vignettes had CPU time 5.4 times elapsed time
```

File: DEV/scripts/ppr_binnenveld.R [hunk 1]
```diff
@@ -9,5 +9,5 @@
   binnenveld <- fread('dev/data/binnenveld.csv')
 
   # save file
-  usethis::use_data(binnenveld)
+  usethis::use_data(binnenveld, version = 3, overwrite = TRUE, compress = 'xz')
```
Comment (gerardhros) [inline DEV/scripts/ppr_binnenveld.R:13]: if needed you can make this set smaller. its only used for illustration purposes in the vignettes.

**Review by gerardhros — CHANGES_REQUESTED** [review]
Thanks. Adding these data files and ppr files are fine.
Note that the package as such is expanding in size. But that is fine.
Note that i saw in the csv files that in some cases a missing value is considered empty and sometimes there is NA in the csv. Please check.

**Comment by SvenVw** [comment]
As this PR does not add new tables or functions to the built R package, it is not expected that the package will grow in size.

File: DEV/scripts/ppr_binnenveld.R (hunk 1 above)
Comment (SvenVw) [reply DEV/scripts/ppr_binnenveld.R:13]: The size of this dataset 37kb, which seems not a of big concern to me. The table waterstress.obic is 279kb. The arguments were added to make those settings explicit and to benefit from better serialization and compression

**Review by gerardhros — APPROVED** [review]

**Pull request merged**